easybuild/easyconfigs/m/mstore/mstore-0.2.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
```

Comment:
@lexming please reinstate the author attribution

Reply:
I know you've made some changes, but still...

Reply:
fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16760/commits/8357b48c2f784b23a7b4d414c648d78ec83f5edf